drake/systems/plants/collision/test/FCLModelTest.cpp

```diff
@@ -0,0 +1,518 @@
+// Get an example of Bullet Collision detection working.
```

Comment:
The code compares results form both Bullet and FCL, but FCL in the comment is more accurate.
This test is a work in progress and will eventually be made into real test.